code/game/gamemodes/miniantags/morph/morph.dm

```diff
@@ -100,6 +100,7 @@
 	visible_message("<span class='warning'>[src] suddenly twists and changes shape, becoming a copy of [target]!</span>", \
 					"<span class='notice'>You twist your body and assume the form of [target].</span>")
 	appearance = target.appearance
+	alpha = 255	//fucking chameleons
```

Comment:
This should be alpha = max(alpha, 150)
Or so, at least, so that semitransparent objects aren't too affected by this.